RELEASE_NOTES.md

```diff
@@ -2,28 +2,34 @@
-  - `electrical_components.Fuse` has been removed.
-  - `InverterType.INVERTER_TYPE_WIND_TURBINE` has been removed.
-  - `ElectricalComponentConnections.source_component_id` has been renamed to `ElectricalComponentConnections.source_electrical_component_id`
-  - `ElectricalComponentConnections.destination_component_id` has been renamed to `ElectricalComponentConnections.destination_electrical_component_id`
-  - `ElectricalComponentStateSnapshot.component_id` has been renamed to `ElectricalComponentStateSnapshot.electrical_component_id`
-  - `microgrid.MicrogridComponentIds` has been renamed to `microgrid.MicrogridElectricalComponentIds`.
-  - `MicrogridComponentIDs.component_ids` has been renamed to `MicrogridElectricalComponentIds.electrical_component_ids`.
-  - `electrical_components.VoltageTransformer` has been renamed to `electrical_components.PowerTransformer`, to make the name more consistent with the underlying concept.
-  - `ElectricalComponentCategorySpecificInfo.kind.voltage_transformer` has been renamed to `ElectricalComponentCategorySpecificInfo.kind.power_transformer`, to make the name more consistent with the underlying concept.
-  - `ElectricalComponentCategory.ELECTRICAL_COMPONENT_CATEGORY_VOLTAGE_TRANSFORMER` has been renamed to `ElectricalComponentCategory.ELECTRICAL_COMPONENT_CATEGORY_POWER_TRANSFORMER`, to make the name more consistent with the underlying concept.
-  - The `type` package has been renamed to `types`, to avoid using reserved keywords in programming languages.
+    + `electrical_components.Fuse`
+    + `InverterType.INVERTER_TYPE_WIND_TURBINE`
 
+- Renamed several symbols to increase consistency and clarity:
 
-## Bug Fixes
+    + `microgrid`:
 
-<!-- Here goes notable bug fixes that are worth a special mention or explanation -->
+        * `MicrogridComponentIds` to `MicrogridElectricalComponentIds`
+        * `MicrogridComponentIDs.component_ids` to `MicrogridElectricalComponentIds.electrical_component_ids`
+
+    + `electrical_components`:
+
+        * `ElectricalComponentConnections.source_component_id` to `ElectricalComponentConnections.source_electrical_component_id`
+        * `ElectricalComponentConnections.destination_component_id` to `ElectricalComponentConnections.destination_electrical_component_id`
+        * `ElectricalComponentStateSnapshot.component_id` to `ElectricalComponentStateSnapshot.electrical_component_id`
+        * Transformer-related terms are renamed to align them with power transformers, which are more commonly used in electrical engineering:
+            * `electrical_components.VoltageTransformer` to `electrical_components.PowerTransformer`
+            * `ElectricalComponentCategorySpecificInfo.kind.voltage_transformer` to `ElectricalComponentCategorySpecificInfo.kind.power_transformer`
+            * `ElectricalComponentCategory.ELECTRICAL_COMPONENT_CATEGORY_VOLTAGE_TRANSFORMER` to `ElectricalComponentCategory.ELECTRICAL_COMPONENT_CATEGORY_POWER_TRANSFORMER`
+
+    + `types`:
```

Comment:
This should probably be `type`, we can fix it in the GitHub release notes at least.